doc/content/getting-started/aws/ecs/database-operations/_index.md

```diff
@@ -167,3 +167,54 @@ tti-lw-stack,is-db,migrate
 ```
 {{</ tabs/tab >}}
 {{</ tabs/container >}}
+
```

Comment:
I know it's not related but could you also add the `dcs-db migrate` as part of this PR? Saves me time from creating a new one.